Mathlib/Algebra/FreeMonoid/Symbols.lean

```diff
@@ -0,0 +1,35 @@
+/-
+Copyright (c) 2024 Hannah Fechtner. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Hannah Fechtner
+-/
+
+import Mathlib.Algebra.FreeMonoid.Basic
+import Mathlib.Data.Finset.Basic
+
+/-!
+# The finite set of symbols in a FreeMonoid element
+
+This is separated from the main FreeMonoid file, as it imports the finiteness hierarchy
+-/
+
+variable {α : Type*} [DecidableEq α]
+
+open FreeMonoid
```

Comment:
I'm going to guess that this should be a `namespace`, rather than `open`.

As a consequence, the `to_additive` of `symbols` would have to be defined separately (I think that only theorems/lemmas are auto-generated by `to_additive`).

Reply:
aha! thank you so much, that was my error when separating this out from the main file